### ISSUE TITLE
overfitting on small dataset?

### ISSUE THREAD

**Issue by rainsoulsrx** [cswry/SeeSR#14]
Hi, I use 500 human face images(only contain 6 id) to train this model, but when I test the trained data, the result is strange, I test using A people, but the result is B people,why?

**Comment by cswry** [comment]
Helo, this is an interesting phenomenon, as we haven't encountered this issue during training on a large-scale dataset. Can you provide detailed training specifics?

**Comment by rainsoulsrx** [comment]
> Helo, this is an interesting phenomenon, as we haven't encountered this issue during training on a large-scale dataset. Can you provide detailed training specifics?

Yes. I use about 500 pair datases, phone image and corresponding camera image after Optical flow registration. the dataset only contain  6 ids. training config I use what you provide, just replace the training data
